bitnami/keycloak/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: keycloak
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/keycloak
   - https://github.com/keycloak/keycloak
-version: 9.7.2
```

Comment:
Could you bump the minor version? It is adding a non-breaking feature

Reply:
Of course. Fixed that!